libs/wire-api/src/Wire/API/MakesFederatedCall.hs

```diff
@@ -0,0 +1,135 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# LANGUAGE OverloadedLists #-}
+
+module Wire.API.MakesFederatedCall
```

Comment:
Doesn't this module belong to the `wire-api-federation` library instead?

Reply:
Spiritually, but the package topography doesn't allow it to be so: `wire-api-federation` depends on `wire-api`, but `wire-api` is where all the APIs are defined.